internal/provider/general_api_models.go

```diff
@@ -36,6 +36,16 @@ type BaseDetailAPIModelWithOrg struct {
 	Organization  int64                 `json:"organization"`
 }
 
+type BaseEdaAPIModel struct {
+	Name string `json:"name"`
+	Id   int64  `json:"id"`
+	URL  string `json:"url"`
+}
```

Comment:
Similar feedback to the `Schema` in base_eda_datasource.go. Do we expect everything under the EDA API to have these fields?